draft-ietf-cdni-ci-triggers-rfc8007bis.xml

```diff
@@ -2883,7 +2889,125 @@ https://video.example.com/k/movie1/4/013.ts
          ]]></artwork>
           </figure>
         </section>
-      </section>
+	</section>
+	<section anchor="execution-policy" title="Execution Extension">
```

Comment:
I was a little surprised to not see this named "ExecutionPolicy Extension", if only to be consistent with how other extensions are named (and with the anchor name).

Reply:
Would not it be cool to leave it as is? :)
Fixed